### ISSUE TITLE
Add user profile page

### ISSUE THREAD

**Issue by vprusso** [unitaryfoundation/metriq-app#188]
Related: https://github.com/unitaryfund/metriq-app/issues/163

A user profile page could be similar to the Scirate user page, as @nathanshammah pointed out, i.e.:
https://scirate.com/118

The analogous URL for us would be https://metriq.info/x where "x" is the unique ID corresponding to the user in the user DB. 

Other features that appear on the Scirate user page would also be interesting to include. For instance, more details about the user including their current affiliation, location, website, date joined, etc. 

Including a tabbed view such as:
![Screen Shot 2021-10-04 at 7 18 37 AM](https://user-images.githubusercontent.com/1562214/135842555-78646cb6-346c-4257-8d9d-f3948e173ca4.png)

would be useful as well. Not all tabs, in this case, would be applicable for metriq, but tabs pertaining to "activity", "likes", etc. would be. If we decide to add comments at a later stage, we could include the retention of this as well.

**Comment by vprusso** [comment]
Indeed! Closing.

**Comment by cosenal** [comment]
@vprusso @nathanshammah Has this feature ever been implemented? I can't find it on the website.

**Comment by vprusso** [comment]
No, this feature is not implemented. Not sure if we still want it, but it's not there now.

**Comment by cosenal** [comment]
I would love this feature. In fact now it's a weird UX that the username is a link in many pages, but it's not clickable (see screenshot). I expect that one could click it and go to a user profile page. Reopening this issue.

![Image](https://github.com/user-attachments/assets/648b46ae-4270-487e-9444-70471026285f)

**Comment by scssandanayake** [comment]
hello! @cosenal, I would like to work on this task 'Add user profile page'

**Comment by cosenal** [comment]
hi @scssandanayake, do you have any question about the feature? Did you manage to set up a dev environment?

**Comment by WingCode** [comment]
Hi @cosenal,
Could you please assign me to the issue as well, so that the UH scraper can pick it up for the bounty?
Thank you!

**Comment by cosenal** [comment]
> Hi [@cosenal](https://github.com/cosenal), Could you please assign me to the issue as well, so that the UH scraper can pick it up for the bounty? Thank you!

Done!
